test/unit/uk/gov/hmrc/exports/models/declaration/ExportsDeclarationSpec.scala

```diff
@@ -39,205 +39,15 @@ class ExportsDeclarationSpec extends WordSpec with MustMatchers {
-      |    "goodsLocation": {
-      |      "country": "United Kingdom",
-      |      "typeOfLocation": "A",
-      |      "qualifierOfIdentification": "U",
-      |      "identificationOfLocation": "FXTFXTFXT",
-      |      "additionalIdentifier": "123"
-      |    },
-      |    "officeOfExit": {
-      |      "officeId": "GB000054",
-      |      "circumstancesCode": "No"
-      |    },
-      |    "supervisingCustomsOffice": {
-      |      "supervisingCustomsOffice": "GBLBA001"
-      |    },
-      |    "warehouseIdentification": {},
-      |    "inlandModeOfTransportCode": {
-      |      "inlandModeOfTransportCode": "1"
-      |    }
-      |  },
-      |  "items": [
-      |    {
-      |      "id": "b12agbfd",
-      |      "sequenceId": 1,
-      |      "procedureCodes": {
-      |        "procedureCode": "1040",
-      |        "additionalProcedureCodes": [
-      |          "000"
-      |        ]
-      |      },
-      |      "fiscalInformation": {
-      |        "onwardSupplyRelief": "No"
-      |      },
-      |      "statisticalValue": {
-      |        "statisticalValue": "1000"
-      |      },
-      |      "commodityDetails": {
-      |        "combinedNomenclatureCode": "46021910",
-      |        "descriptionOfGoods": "Straw for bottles"
-      |      },
-      |      "dangerousGoodsCode": {
-      |        "dangerousGoodsCode": "1234"
-      |      },
-      |      "cusCode": {
-      |        "cusCode": "12345678"
-      |      },
-      |      "taricCodes": [],
-      |      "nactCodes": [],
-      |      "packageInformation": [
-      |        {
-      |          "typesOfPackages": "PK",
-      |          "numberOfPackages": 10,
-      |          "shippingMarks": "RICH123"
-      |        }
-      |      ],
-      |      "commodityMeasure": {
-      |        "supplementaryUnits": "10",
-      |        "netMass": "500",
-      |        "grossMass": "700"
-      |      },
-      |      "additionalInformation": {
-      |        "items": [
-      |          {
-      |            "code": "00400",
-      |            "description": "EXPORTER"
-      |          }
-      |        ]
-      |      },
-      |      "documentsProducedData": {
-      |        "documents": [
-      |          {
-      |            "documentTypeCode": "C501",
-      |            "documentIdentifier": "GBAEOC717572504502811"
-      |          }
-      |        ]
-      |      }
-      |    }
-      |  ],
-      |  "totalNumberOfItems": {
-      |    "totalAmountInvoiced": "56764",
-      |    "exchangeRate": "1.49",
-      |    "totalPackage": "1"
-      |  },
-      |  "previousDocuments": {
-      |    "documents": [
-      |      {
-      |        "documentCategory": "Y",
-      |        "documentType": "IF3",
-      |        "documentReference": "101SHIP2"
-      |      }
-      |    ]
-      |  },
-      |  "natureOfTransaction": {
-      |    "natureType": "1"
-      |  }
-      |}""".stripMargin
-
   val declarationVersion2: String =
```

Comment:
You can remove `2` from the name. There is one version of the declaration right now